discord/interactions.py

```diff
@@ -321,6 +322,7 @@ async def original_message(self):
         InteractionMessage
             The original interaction response message.
         """
+        warnings.warn("Use 'Interaction.original_response()' instead.", DeprecationWarning)
```

Comment:
I think it would be better to use this format for the message: `"{old_method} is deprecated, use {new_method} instead."`